.eslintrc.js

```diff
@@ -924,6 +924,23 @@ module.exports = {
       ],
       rules: {
         '@kbn/telemetry/event_generating_elements_should_be_instrumented': 'error',
+      },
+    },
+    {
+      files: [
```

Comment:
What is this config about?

Reply:
In order to use a custom rule, you need a custom package that exports the rules. 
In this case that's `packages/kbn-eslint-plugin-i18n`. 

Then to add it to the ESLint config, you need to add it to `eslintrc.js` in two places: 
in `plugins` at the top. That allows ESLint to 'see' the rules that are in your custom package.

Then further down in the file you can enable the rules by the name you exported it.  

So in this block I add the rules on all `.tsx` files in the paths associated with Observability apps.